CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ The API consists of all public Kotlin types from `com.atlassian.performance.tool
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/ssh/compare/release-2.4.3...master
 
+### Changed
+- Bump ssh-ubuntu library to version without vulnerable log4j
+
```

Comment:
```suggestion
```
It doesn't change anything in the API (including POM). It's only affecting the local dev loop and CI, not consumers.